compiler/ballerina-parser/src/test/java/io/ballerinalang/compiler/parser/test/ParserTestUtils.java

```diff
@@ -391,6 +399,8 @@ private static SyntaxKind getNodeKind(String kind) {
                 return SyntaxKind.OPEN_BRACE_PIPE_TOKEN;
             case "CLOSE_BRACE_PIPE_TOKEN":
                 return SyntaxKind.CLOSE_BRACE_PIPE_TOKEN;
+            case "PIPE_TOKEN":
+                return SyntaxKind.PIPE_TOKEN;
```

Comment:
`|` had been declared under `separators` in LexerTermianls.java. So I added it into the same category here also. 
Let me knw if it should be moved to `Arithmetic operators` category.